src/Prometheus.Client.MetricPusher.HostedService.Tests/Prometheus.Client.MetricPusher.HostedService.Tests.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Is ImplicitUsings works for netappcore3.1 or net5.0?

Reply:
Removed